### PR TITLE
Update PoE tutorial to FRAME v2

### PR DESCRIPTION
**Pull request by sacha-l**
This updates the "build a dapp" / PoE tutorial to FRAME v2. 

@NukeManDan @jimmychu0807 please double check on your end. Works on mine but ... verify first ;)

This will be followed by updating the solutions branch in the node-template repository.

### CODE REVIEW

**Comment by sacha-l** [comment]
Updated the solution too: https://github.com/substrate-developer-hub/substrate-node-template/commit/a097aceef4c988d5e4ebb3bfa297f3b28192f20a

File: docs/tutorials/build-a-dapp/pallet.md [hunk 1]
```diff
@@ -311,16 +276,12 @@ decl_module! {
             Proofs::<T>::remove(&proof);
 
             // Emit an event that the claim was erased.
-            Self::deposit_event(RawEvent::ClaimRevoked(sender, proof));
+            Self::deposit_event(Event::ClaimRevoked(sender, proof));
+
+            Ok(().into())
```
Comment (jimmychu0807) [inline docs/tutorials/build-a-dapp/pallet.md:281]: Just aligns the space/tab back.

**Review by jimmychu0807 — APPROVED** [review]
Overall I am happy with the PoE update. I just made suggestions on some minor wording updates and aligning spacings in the code shown in md files.

**Comment by NukeManDan** [comment]
Did a full run, updated the solutions here: https://github.com/substrate-developer-hub/substrate-developer-hub.github.io/pull/1008/
@sacha-l take a look, I added a few clarifying things I hope are helpful. 

If you like, merge that into this branch, and then we can merge this PR into source 😁

**Review by NukeManDan — APPROVED** [review]

**Pull request merged**